BothamNetworking/BothamAPIClient.swift

```diff
@@ -115,6 +115,15 @@ public class BothamAPIClient {
         return interceptedResponse
     }
 
+    private func mapNSErrorToBothamAPIClientError(error: NSError) -> BothamAPIClientError {
```

Comment:
We are already requesting to mapping HTTPRequest and HTTPResponse why not the Error.